std/hash/keccak256/keccak_test.go

```diff
@@ -0,0 +1,53 @@
+package keccak
+
+import (
+	"github.com/consensys/gnark-crypto/ecc"
+	"github.com/consensys/gnark/backend"
+	"github.com/consensys/gnark/frontend"
+	"github.com/consensys/gnark/test"
+	"golang.org/x/crypto/sha3"
```

Comment:
we better use keccak from go-ethereum.

"github.com/ethereum/go-ethereum/crypto"

crypto.Keccak256()

Reply:
done